libs/client/my-stuff/src/lib/components/content-preview/content-preview.component.ts

```diff
@@ -33,9 +33,13 @@ export class ContentPreviewComponent {
         private stuff: MyStuffService,
         public stuffQuery: MyStuffQuery,
         private router: Router,
-        private alerts: AlertsService
+        private alerts: AlertsService,
     ) {}
 
+    ngOnInit() {
+        this.stuffQuery.current$.subscribe((x) => console.log(x));
+    }
```

Comment:
If this is just logging, remember to remove later

Reply:
yup yup!